src/ripple/protocol/Feature.h

```diff
@@ -74,7 +74,7 @@ namespace detail {
 // Feature.cpp. Because it's only used to reserve storage, and determine how
 // large to make the FeatureBitset, it MAY be larger. It MUST NOT be less than
 // the actual number of amendments. A LogicError on startup will verify this.
-static constexpr std::size_t numFeatures = 68;
+static constexpr std::size_t numFeatures = 69;
```

Comment:
Just a reminder that this needs to be incremented when the other amendment is merged. We won't get a merge conflict, since they change to the same number.